app/controllers/albums.js

```diff
@@ -0,0 +1,29 @@
+const albumsService = require('../services/albums');
+
+function albums(req, res) {
+  albumsService
+    .albums()
+    .then(albumsData => res.status(200).json({ status: 200, data: albumsData }))
+    .catch(error =>
+      res
```

Comment:
Check [this](https://expressjs.com/en/guide/error-handling.html) link for error handling in controller, also check the middleware of errors for map the error with a message and a status code